firmware/Core/Src/telecommands/flash_telecommand_defs.c

```diff
@@ -263,7 +263,7 @@ uint8_t TCMDEXEC_flash_erase(const char *args_str, TCMD_TelecommandChannel_enum_
     snprintf(
         &response_output_buf[strlen(response_output_buf)],
         response_output_buf_len - strlen(response_output_buf) - 1,
-        " Successfully erased page at address %lu on chip %d.\n", 
+        "Successfully erased block containing page address %lu on chip %d.\n", 
```

Comment:
"Page address" or "page number"? A page address would be the full address of the first byte of the page, whereas the page number would be `page_address / page_size`. 

Double-check that the variable name matches the description.

Reply:
you're right, the proper description should be page number. I'll also changed the name of the variable to properly match this description.